cleanlab/rank.py

```diff
@@ -252,11 +253,15 @@ def get_label_quality_ensemble_scores(
     adjust_pred_probs : bool, optional
       `adjust_pred_probs` in the same format expected by the :py:func:`get_label_quality_scores <cleanlab.rank.get_label_quality_scores>` function.
 
-    weight_ensemble_members_by : {"uniform", "accuracy"}, default="accuracy"
+    weight_ensemble_members_by : {"uniform", "accuracy", "custom"}, default="accuracy"
       Weighting scheme used to aggregate scores from each model:
 
       - "uniform": take the simple average of scores
       - "accuracy": take weighted average of scores, weighted by model accuracy
+      - "custom": take weighted average of scores using custom weights that the user passes to the custom_weights parameter.
+
+    custom_weights : np.array, default=None
+      Weights used to aggregate scores from each model if weight_ensemble_members_by="custom".
```

Comment:
specify length of this array must match the number of models, i.e. `len(pred_probs_list)`.
Would be good to have explicit check that this actually matches too and raise ValueError if not

Reply:
Thanks, done! I also added an assertion check.